dask_kubernetes/tests/test_async.py

```diff
@@ -619,8 +619,9 @@ async def test_repr(cluster):
 
 
 @pytest.mark.asyncio
-async def test_escape_username(pod_spec, ns, auth, monkeypatch):
-    monkeypatch.setenv("LOGNAME", "foo!._")
+@pytest.mark.parametrize("name", ["foo!._", "Foo!._"])
```

Comment:
I think parametrizing the tests here is unnecessary. Changing it to uppercase would be acceptable.

Reply:
Thanks. Yes, I think it makes sense to keep just one test case. Reverted the change.